tests/test_file.py

```diff
@@ -72,7 +70,7 @@ def test_upload_file(client: ThanoSQL, basic_table_name: str):
         db_commit=True,
         table=basic_table_name,
         column=column_name,
-        dir=dir,
+        dir=dir_name,
     )
     assert res["data"]["table_name"] == basic_table_name
```

Comment:
```suggestion
        dir=dir_name,
    )
    assert res["data"]["file_path"] == f"drive/{dir_name}/{file_name}"
    assert res["data"]["table_name"] == basic_table_name
```
dir이 정상적으로 작동하는지 위 케이스 추가해줄래요?

Reply:
좋을 것 같아요~